src/components/Filter/LessonFilter.js

```diff
@@ -36,15 +37,17 @@ const LessonFilter = ({filterGroupKeys, isStudentMode, t}) => {
       <Col xsHidden>
         <Panel {...{header, bsStyle}}>
           <ListGroup fill>
-            {filterGroups}
+            <RadioButtons/>
+            {playlists ? (language === 'nb' ? null : filterGroups) : filterGroups}
           </ListGroup>
         </Panel>
       </Col>
       {/*Filter mobile*/}
       <Col smHidden mdHidden lgHidden>
-        <CollapsiblePanel initiallyExpanded={false} {...{header, bsStyle}}>
+        <CollapsiblePanel initiallyExpanded={true} {...{header, bsStyle}}>
           <ListGroup fill>
-            {filterGroups}
+            <RadioButtons/>
+            {playlists ? (language === 'nb' ? null : filterGroups) : filterGroups}
```

Comment:
I `mapStateToProps` (som jo er en selector) kunne du jo lage `showRadiobuttons` og `showFilterGroups`, i stedet for å sende inn language og playlists rått...

Reply:
Skjønte ikke helt...

Reply:
I `mapStateToProps` kan du ha:
```
showRadiobuttons: state.language === 'nb',
showFiltergroups: !state.playlists || state.language !== 'nb',
 ```
og så trenger du ikke `playlists` eller `language` i props.